### PR TITLE
GCE Compute Profile and Load Zones support added

### PR DESCRIPTION
**Pull request by jyejare**
Added support for GCE Compute Resource for :

- [x] Load Zones
- [x] Create Compute Profile

### CODE REVIEW

File: airgun/views/computeresource.py [hunk 1]
```diff
@@ -85,6 +85,14 @@ class GCEProviderForm(View):
         certificate_path = TextInput(id='compute_resource_key_path')
         load_zones = Text("//*[contains(@id,'test_connection_button')]")
 
+        @View.nested
```
Comment (oshtaier) [inline airgun/views/computeresource.py]:
```
        load_zones = Text("//*[contains(@id,'test_connection_button')]")

        @View.nested
        class zone(View):
            load_zones = Text("//a[contains(@id,'test_connection_button')]")
            ...
```

looks as code duplication

Comment (jyejare) [reply airgun/views/computeresource.py]: Done!

Result Passed after change:
```
$ py.test tests/foreman/ui/test_computeresource_gce.py
========================================== test session starts ==========================================
platform linux -- Python 3.6.8, pytest-4.0.2, py-1.7.0, pluggy-0.8.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.4, forked-1.0.2, cov-2.6.1
collecting ... 2019-06-20 16:51:13 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item

================================ 1 passed, 4 warnings in 742.38 seconds =================================
```

**Comment by oshtaier** [comment]
@jyejare ACK
but I cannot merge as I am not on the project anymore and my account is removed from organization

**Comment by jyejare** [comment]
@oshtaier Thanks Alex, sad to see u going! Couldn't attend the last meeting with you on Friday, had doctor appointment and hence I left early.

**Comment by jyejare** [comment]
@mirzal Yeah for test results we can look at the UI test automated using this PR : https://github.com/SatelliteQE/robottelo/pull/7041.

**Comment by oshtaier** [comment]
@jyejare no problem, reach me at any time if you need help. I would gladly do that

**Pull request merged**